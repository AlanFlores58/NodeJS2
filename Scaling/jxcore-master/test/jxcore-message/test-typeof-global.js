// Copyright & License details are available under JXCORE_LICENSE file


var print = function(obj, name) {
  console.log("\ttypeof", name, "=", typeof obj);
};

console.log("\nnamespace:", "globals");
print(Buffer, "Buffer");
print(GLOBAL, "GLOBAL");
print(clearInterval, "clearInterval");
print(clearTimeout, "clearTimeout");
print(console, "console");
print(global, "global");
print(jxcore, "jxcore");
print(process, "process");
print(root, "root");
print(setImmediate, "setImmediate");
print(setInterval, "setInterval");
print(setTimeout, "setTimeout");
