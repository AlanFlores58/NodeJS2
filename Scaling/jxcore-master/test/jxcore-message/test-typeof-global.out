
namespace: globals
	typeof Buffer = function
	typeof GLOBAL = object
	typeof clearInterval = function
	typeof clearTimeout = function
	typeof console = object
	typeof global = object
	typeof jxcore = object
	typeof process = object
	typeof root = object
	typeof setImmediate = function
	typeof setInterval = function
	typeof setTimeout = function
